README.md

```diff
@@ -14,6 +14,8 @@ Next.js, Urql, Apollo, Prisma, PostgreSQL
 - Install dependencies in `/ui`
   - `yarn`
 - In /ui, copy `.env.local.default` to `.env.local`
+- In /ui, copy and paste DATABASE_URL string from `.env.development` to `.env.local`
```

Comment:
I don't think this should be needed? Both files are loaded.

But ah on checking `.env.local.default` we have `DATABASE_URL=""` in it which clears the variable. I think a better solution is to just remove that line.